examples/broken/correlation_alignment_example.py

```diff
@@ -16,6 +18,71 @@
 from collections import OrderedDict
 from numpy.random import seed
 
+
+# {{{ Define the frequency mask function and the ph cyc mask
+def frq_mask(s, signal_pathway, direct="t2", indirect="repeats"):
+    """Generates a mask that is nonzero along frequencies only over the
+    bandwidth of the signal
+        Parameteres
+        ===========
+        s: nddata
+            data that the mask is applied to
+        signal_pathway: dict
+            This is the default pathway that will be nonzero since this is
+            where your signal is.
+        direct: str
+            Direct dimension
+        indirect: str
+            The indirect dimension (does not include phase cycling dimensions)
+
+    Returns
+        =======
+        s: nddata
+            copy of data with the mask applied
+    """
+    # Just for consistency we will start with the frequency domain in the phase
+    # cycling dimension
+    assert s.get_ft_prop(direct), "You must be in the frequency domain!"
+    for phnames in signal_pathway.keys():
+        assert not s.get_ft_prop(phnames), (
+            str(phnames) + "must NOT be in coherence domain!"
+        )
+    signal_keys = list(signal_pathway)
+    signal_values = list(signal_pathway.values())
+    # we want to leave the original s unchanged and return a copy
+    for_mask = s.C
+    for_mask.ft(list(signal_pathway))
```

Comment:
you should not FT.  You should just assume the data will be provided in frequency (and coherence) domain